make/DynamoRIOConfig.cmake.in

```diff
@@ -245,13 +245,6 @@
 # should have a "_DR_" prefix.  Public functions and variables should have DynamoRIO
 # in the name.
 
-if ("${CMAKE_VERSION}" VERSION_EQUAL "3.0" OR
-    "${CMAKE_VERSION}" VERSION_GREATER "3.0")
-  # XXX i#1557: update our code to satisfy the changes in 3.x
-  cmake_policy(PUSH)
-  cmake_policy(SET CMP0026 OLD)
```

Comment:
I still see uses of the LOCATION property in this file:
```
make/DynamoRIOConfig.cmake.in:  get_target_property(client_path ${target} LOCATION${_DR_location_suffix})
make/DynamoRIOConfig.cmake.in:      get_target_property(libpath dynamorio LOCATION${_DR_location_suffix})
make/DynamoRIOConfig.cmake.in:      get_target_property(libpath ${extname} LOCATION${_DR_location_suffix})
```

Some of these may hit the imported-target issue.

Reply:
Yes. How can I test this best? Scan existing client to use this API?

Reply:
samples_proj uses this -- why didn't it hit problems with your removal of the policy yet leaving LOCATION in there?

Reply:
Checking..

Reply:
Doesn't look like samples_proj is hitting any of those LOCATION calls, though calling some of the functions. That's prob. why there was also no policy error. If LOCATION was part of the target(s) for external clients, why do you think that the <VAR>_OUTPUT_DIRECTORY would not (and be a problem)? I do think that I need to pass in the location_suffix into the DynamoRIO_get_full_path function, either as location_suffix or e.g. as _DR_location_suffix.

Reply:
> If LOCATION was part of the target(s) for external clients, why do you think that the VAR_OUTPUT_DIRECTORY would not (and be a problem)?

Imported targets explicitly set an IMPORTED_LOCATION property (see e.g. DynamoRIO-Linux-7.0.0-RC1/cmake/DynamoRIOTarget32-relwithdebinfo.cmake) which I assume ends up in LOCATION, whereas those generated import cmake files do not set any *OUTPUT_DIRECTORY, so its value may come from the importing project's globals.  The Dr. Memory experiment points in that direction.  I would suggest experimenting with the public samples project to see what the values are of these properties.